### ISSUE TITLE
Request to update the benchmark table

### ISSUE THREAD

**Issue by csukuangfj** [guillaumekln/faster-whisper#425]
The [benchmark table](https://github.com/guillaumekln/faster-whisper#small-model-on-cpu) contains only `openai/whisper`, `whisper.cpp`, and `faster-whisper`.

Recently, we have added support for whisper models to [sherpa-onnx](http://github.com/k2-fsa/sherpa-onnx), which uses [onnxruntime](https://github.com/microsoft/onnxruntime) as the backend and supports both CPU and NVIDIA GPU. It also provides Python APIs and APIs for other languages.

Since the underlying implementation of [sherpa-onnx](http://github.com/k2-fsa/sherpa-onnx) is also in C++, it would be great if the benchmark table also includes [sherpa-onnx](http://github.com/k2-fsa/sherpa-onnx).

**Comment by guillaumekln** [comment]
Hi,

Does this project implement the complete Whisper algorithm including the beam search, the condition on the previous text, the temperature fallback, etc.?

**Comment by csukuangfj** [comment]
`sherpa-onnx` implements only greedy search for whisper models. I suggest that we also add a column for greedy_search in the benchmark table.

Recognition results obtained by greedy search are also of high accuracy.

**Comment by guillaumekln** [comment]
Yes the table compares the transcription time, but each tool is running the complete Whisper algorithm as implemented in this reference file:

https://github.com/openai/whisper/blob/main/whisper/transcribe.py

For a fair comparison, new tools should implement the same algorithm and use the same beam size for decoding. Otherwise we would need to add a new table that compares a different configuration.
